scripts/linters/third_party_typings_linter.py

```diff
@@ -0,0 +1,158 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Lint checks for third party type definitions."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import json
+import os
+import sys
+
+import python_utils
+from . import linter_utils
+
+_MANIFEST_JSON_FILE_PATH = os.path.join(os.getcwd(), 'manifest.json')
+_PACKAGE_JSON_FILE_PATH = os.path.join(os.getcwd(), 'package.json')
+_TYPE_DEFS_FILE_EXTENSION_LENGTH = len('.d.ts')
+
+THIRD_PARTY_LIBS = [
```

Comment:
@Hudda, Do we have the infrastructure to have a unit test for this? (Though I think the unit test for the specif linter file will be easy can we do the integration testing?)

@nishantwrp, Can you please write a unit test for this file?

Reply:
@DubeySandeep Yep, We need unit tests for this file. But I have no idea about writing integration tests. Can you please point me to the right resources and if possible an example in the codebase.

Reply:
I couldn't find spec for other lint files. Can you refer me a file which i can refer for writing tests,

Reply:
I don't think we have a unit test file for any linter but we are planning to have it for all the lint test files. You can check the test file for the release script, they will be helpful!

> But I have no idea about writing integration tests.
@Hudda, By integration test I mean running pre_commit_linter test this new test i.e, testing the integration of this typedef check with the pre-commit linter. I think we can have it later but I think a unit test for this new file should be mandatory!

Reply:
@DubeySandeep Okay then, let @nishantwrp write a unit test for this file and I will add the integration tests later.

Reply:
Sounds good!

Reply:
done